CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CHANGELOG
 
+### 2.2.4
+- Fix - Catch KlaviyoRateLimitException properly
```

Comment:
```suggestion
- Fix - Instantiate KlaviyoRateLimitException properly
```

Reply:
Or initialize or throw or raise

Reply:
the response is 
{"detail":"Request was throttled. Expected available in 53.0 seconds."}